src/app/shared/components/number-stepper/style.scss

```diff
@@ -31,8 +31,7 @@ input[type='number']::-webkit-outer-spin-button {
     margin: 4px 0 0;
 
     &.left {
-      border-bottom-right-radius: 0;
-      border-top-right-radius: 0;
+      border-radius: 0;
```

Comment:
Isn't that default?

Reply:
no it's not if i remove it all the corner will have a radius 
![Screenshot from 2022-03-10 12-21-16](https://user-images.githubusercontent.com/85109141/157631354-41bdda57-8b27-4665-95e5-4f2848dc1aba.png)